### PR TITLE
Core/Misc: update g3dlite lib

### PR DESCRIPTION
**Pull request by Viste**
## CHANGES PROPOSED:
-  update g3dlite lib because there's no reason to use the 2013 version in 2020.
- also fix build on gcc/g++ 9+
## TESTS PERFORMED:
- build on linux log: https://paste2.org/18z9nPwe
## HOW TO TEST THE CHANGES:
lol build ? make new mmaps vmaps?
## Target branch(es):
- [x] Master

## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by FrancescoBorzi** [comment]
> HOW TO TEST THE CHANGES:
> lol build ? make new mmaps vmaps?

I'd say also a quick test in game it's needed, not only building

**Comment by Viste** [comment]
sad story but as always all needs only quests

**Comment by Viste** [comment]
its not a rage quit its just not needed by anyone lol )

**Comment by pklloveyou** [comment]
Tested, no problems

**Comment by FrancescoBorzi** [comment]
@pklloveyou thanks for your feedback, however this PR can't be reopened because @Viste deleted his branch.

I'll never understand what's the benefit of deleting branches that are not being merged yet.

**Comment by Viste** [comment]
@FrancescoBorzi special for you :)

**Comment by FrancescoBorzi** [comment]
@Viste thank you. But build fails :(

**Comment by Viste** [comment]
@FrancescoBorzi fixed, but cheks fails because of dry-run test on locale krKR its not my problem :/

File: data/sql/updates/db_world/2020_04_21_00.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+SELECT sql_rev INTO OK FROM version_db_world WHERE sql_rev = '1518477962308987200'; IF OK <> 'FALSE' THEN LEAVE proc; END IF;
+--
+-- START UPDATING QUERIES
+--
+
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1518477962308987200');
+
+DELETE FROM `item_loot_template` WHERE `Entry` = 54537 AND `Item` = 49426;
+INSERT INTO `item_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
+(54537, 49426, 0, 100, 0, 1, 0, 2, 2, '');
```
Comment (FrancescoBorzi) [inline data/sql/updates/db_world/2020_04_21_00.sql:21]: how is this related to `g3dlite` and why this is not using the pending import? I guess it's a mistake?

Comment (Viste) [reply data/sql/updates/db_world/2020_04_21_00.sql:21]: i think its merge shit >_< w8 few minutes

**Comment by Viste** [comment]
@FrancescoBorzi i fix this merge shhiiiit

**Comment by FrancescoBorzi** [comment]
@Kitzunu let's merge this one next

**Comment by Kitzunu** [comment]
Yes
